### PR TITLE
Adding publisher and year published to the papers.

### PR DESCRIPTION
**Pull request by TMoMoreau**
# Describe your changes

- Adding publisher and year published to each of the papers listed in `academic-papers.md`.
- The first one, "IPFS - Content Addressed, Versioned, P2P File System" is simply listed as "Original IPFS white paper", as suggested in a comment from PR #1242.
- I researched the original publisher and date for each one as best I could, let me know if you notice that some information is incorrect.

# Files changed

- \docs\concepts\further-reading\academic-papers.md

# What issue(s) does this address?

- Issue #1247 

# Does this update depend on any other PRs?

No.

## Checklist before requesting a review
- [ ] Passing the beta version of the **Check Markdown links for modified files** check. Action results can be viewed [here](https://github.com/ipfs/ipfs-docs/actions/workflows/action.yml).

## Checklist before merging
- [ ] Passing all required checks (The beta **Check Markdown links for modified files** check is not required)

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build failed...

```

> ipfs-docs@2.0.0 docs:build
> NODE_OPTIONS=--openssl-legacy-provider vuepress build docs

node: --openssl-legacy-provider is not allowed in NODE_OPTIONS
```

**Comment by TMoMoreau** [comment]
This link: https://www.sigops.org/ew-history/1998/papers/mazieres.ps is now a dead-end. The paper can still be found via the wayback machine, though.

**Comment by TMoMoreau** [comment]
Also, filecorgi is reporting that the vuepress build failed, but it builds successfully on my local test build.

**Comment by 2color** [comment]
Nice! Have you considered using quote elements for this, i.e. `>`

**Review by johnnymatthews — APPROVED** [review]
I'd prefer if we didn't use parentheses, but I guess it's a simple way to differentiate between that and the paper content.

File: docs/concepts/further-reading/academic-papers.md [hunk 1]
```diff
@@ -9,24 +9,36 @@ Here are a few papers that are useful for understanding IPFS, whether it be unde
 
 ## [IPFS - Content Addressed, Versioned, P2P File System](https://github.com/ipfs/papers/raw/master/ipfs-cap2pfs/ipfs-p2p-file-system.pdf)
 
+(Original IPFS white paper)
+
 **Benet, Juan**: _The InterPlanetary File System (IPFS) is a peer-to-peer distributed ﬁle system that seeks to connect all computing devices with the same system of ﬁles. In some ways, IPFS is similar to the Web, but IPFS could be seen as a single BitTorrent swarm, exchanging objects within one Git repository. In other words, IPFS provides a high throughput content-addressed block storage model, with content-addressed hyperlinks. This forms a generalized Merkle DAG, a data structure upon which one can build versioned ﬁle systems, blockchains, and even a Permanent Web. IPFS combines a distributed hashtable, an incentivized block exchange, and a self-certifying namespace. IPFS has no single point of failure, and nodes do not need to trust each other._
 
 ## [Design and Evaluation of IPFS: A Storage Layer for the Decentralized Web](https://ipfs.io/ipfs/bafybeid6doxhzck3me366265u3ony6rbuzv7dze7pjuptxeln24b2qvur4?filename=trautwein2022a.pdf)
 
+(Association for Computing Machinery, 2022)
```
[the remark this answers is not in the document]
Comment (TMoMoreau) [reply docs/concepts/further-reading/academic-papers.md:18]: It was published in SIGCOMM '22: Proceedings of the ACM SIGCOMM 2022, which was published by the Association for Computing Machinery.

Screen Shot 2023-01-18 at 2 45 33 PM

https://dl.acm.org/doi/proceedings/10.1145/3544216

**Comment by TMoMoreau** [comment]
Just tested it out, @2color's recommendation to use the quote tag `>` looks better IMO. Win win.

File: docs/concepts/further-reading/academic-papers.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TMoMoreau) [reply docs/concepts/further-reading/academic-papers.md]: The original suggestion was to add "where the paper was published", the information I added in this PR is the **who** published the paper.

**Comment by 2color** [comment]
Pretty minor visual thing, but what do you think about moving it below the paper synopsis/abstract? 

Screen Shot 2023-01-20 at 2 07 40 PM

**Review by 2color — APPROVED** [review]

**Pull request merged**